bleualign/align.py

```diff
@@ -217,10 +217,12 @@ def _outputObjectFromParameter(self, parameter, filename, suffix):
         return outputObject, close_object
 
     #takes care of multiprocessing; calls process() function for each article
-    def mainloop(self):
-      
+    def mainloop(self, vs_debug_mode=False):
       results = {}
-
+      if vs_debug_mode == True: #vscode will raise errors unless multiprocessing is disabled.  
+        multiprocessing_enabled = False
+      else:
+        multiprocessing_enabled = True
```

Comment:
It should put around line 19.
Is `... platform.system() != "Windows":` not working?

Reply:
Well, if I put these 4 lines around line 19, users can't customize this module's behavior via calling function if they do want debugging from vscode. This change by me might not be suitable for most of people who only want use functionality. That is OK if this feature is not merged. But a little suggestion: you can put a note in readme, say, multiprocessing feature can be disabled in source code to make debug in vscode possible.